pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -91,12 +91,13 @@ type InstallationSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	Type             string         `json:"type"`
-	RoutingSubdomain string         `json:"routingSubdomain,omitempty"`
-	MasterURL        string         `json:"masterUrl,omitempty"`
-	NamespacePrefix  string         `json:"namespacePrefix,omitempty"`
-	SelfSignedCerts  bool           `json:"selfSignedCerts"`
-	PullSecret       PullSecretSpec `json:"pullSecret"`
+	Type                 string         `json:"type"`
+	RoutingSubdomain     string         `json:"routingSubdomain,omitempty"`
+	MasterURL            string         `json:"masterUrl,omitempty"`
+	NamespacePrefix      string         `json:"namespacePrefix,omitempty"`
+	SelfSignedCerts      bool           `json:"selfSignedCerts"`
+	PullSecret           PullSecretSpec `json:"pullSecret"`
+	UseExternalResources bool           `json:"useExternalResources"`
```

Comment:
We've actually removed the use of this flag now, we provision resources with cro by default

Reply:
`UseExternalResources`, right? if so I will remove it

Reply:
Yeah :+1: